openai/README.md

```diff
@@ -113,13 +205,118 @@ WARNING:ddtrace.vendor.dogstatsd:Error submitting packet: [Errno 61] Connection
 DEBUG:ddtrace.contrib.openai._logging.py:sent 2 logs to 'http-intake.logs.datadoghq.com'
 ```
 
+
 <!-- xxz tab xxx -->
 <!-- xxx tab "Node.js" xxx -->
 
 **Note**: This setup method does not collect `openai.api.usage.*` metrics. To collect these metrics, also follow the API key setup instructions.
 
 ### Installation
 
+#### Web: Get Account-level Usage and Cost Metrics
```

Comment:
Why is this change repeating the above instructions?

Reply:
In the docs we only have the web section in the python tab
https://docs.datadoghq.com/integrations/openai/?tab=python#setup

Reply:
So I added it to the other tabs. If you believe it was intentional that we only put it in the first language tab let me know to remove it.

Reply:
I see. Here's how we should update the public docs in this case:

- Default tab on public docs when page loads should be "API Key"
- The "API Key" instructions should be update to be the ones in the web section of the python tab. (+ keep listed of collected metrics).
- We should update the tile and the docs from "#### Web: Get Account-level Usage and Cost Metrics" to "#### API Key: Get Account-level Usage and Cost Metrics"
- API Key instructions **should only be visible on the** "API Key" tab

Reply:
> API Key instructions should only be visible on the "API Key" tab

So no Web : in the rest of the language tabs

Reply:
correct, since those instructions are unrelated to python, php, node.js setups. Biggest change here is making sure "API Key" is the first tab in the docs and selected by default on page load.